fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/info/DefaultValue.java

```diff
@@ -40,6 +41,8 @@ public class DefaultValue {
     public static DefaultValue BITMAP_EMPTY_DEFAULT_VALUE = new DefaultValue(ZERO);
     // default "value", "[]" means empty array
     public static DefaultValue ARRAY_EMPTY_DEFAULT_VALUE = new DefaultValue("[]");
+    // default "value", "3.14159265358979323846" means pi
+    public static DefaultValue PI_DEFAULT_VALUE = new DefaultValue("3.14159265358979323846", PI);
```

Comment:
could you explain why this scale is proper. do we need more precision?

Reply:
refer to <math.h>:
```c++
# define M_PI		3.14159265358979323846	/* pi */
```

Reply:
use `M_PIl` in <math.h> maybe better?
```c
# define M_PIl		3.141592653589793238462643383279502884L /* pi */
```

Reply:
I think in legal range, more precision is better as default value. you could try the max precision which double could represent exactly

Reply:
Get.
`M_PI` is enough, because the precision of type double is 15~16 digits. but it is not adequate for computation using long double. in this case, use `M_PIl`.